pavelib/assets.py

```diff
@@ -604,60 +571,11 @@ def process_npm_assets():
     """
     Process vendor libraries installed via NPM.
     """
-    def copy_vendor_library(library, skip_if_missing=False):
```

Comment:
Should this `process_npm_assets` function now call the `process-npm-assets.sh` script? It seems to me that we would need this to be backward compatible.
As a side note, the `openedx-assets.py` script in Tutor calls the `process_npm_assets` function, so we would have to modify it if the function does not call the bash script: https://github.com/overhangio/tutor/blob/b903c69fac95e797532288d4e6f60530faf1db94/tutor/templates/build/openedx/bin/openedx-assets#L117

Reply:
> Should this process_npm_assets function now call the process-npm-assets.sh script? It seems to me that we would need this to be backward compatible.

It does, see [this line of the diff](https://github.com/openedx/edx-platform/pull/31519/files/0be673c541b82748789a60a4f822b528518aad36#diff-8e4387cbbe877b0d2638a138b05ad2d3ecc0b4e3665658612f04601df20cf3ceR578). And yes, goal is backwards-compatibility. 

> As a side note, the openedx-assets.py script in Tutor calls the process_npm_assets function, so we would have to modify it if the function does not call the bash script: https://github.com/overhangio/tutor/blob/b903c69fac95e797532288d4e6f60530faf1db94/tutor/templates/build/openedx/bin/openedx-assets#L117

For what it's worth, I do plan to [change Tutor's openedx-assets.py script to call this bash script directly](https://github.com/kdmccormick/tutor/pull/18/files#diff-2bfcf77b38c5be0f588b5d5af1d4311581dcc6bce5d4abea415234adf73987a2R145)--we will need to do so in order to pass in the `--node-modules=/openedx/node_modules`.  That change is not necessary in order to merge this current PR, though.